### ISSUE TITLE
Please add license in GitHub settings

### ISSUE THREAD

**Issue by fedir** [jquery/jquery#4039]
Please add license in GitHub settings of the repository.

Currently, GitHub could not identify Your project licence, and it's not shown in interface, and not available via API also.

Ref : https://help.github.com/articles/adding-a-license-to-a-repository/

**Comment by mgol** [comment]
Thanks for the report. We do have a license in our repository: https://github.com/jquery/jquery/blob/master/LICENSE.txt. It seems GitHub doesn't like the `txt` extension. I guess we could remove the extension, most repos I've seen don't use one or use `md`.

**Comment by dmethvin** [comment]
The package.json also specifies `license: "MIT"`. I'm fine with removing the `.txt` or changing to `.md`.

**Comment by fedir** [comment]
Hi !

I just get the answer from GitHub support team. Basically, GitHub uses automatic license detector : https://github.com/benbalter/licensee

I just installed it locally and get following results :

```
bundle exec bin/licensee detect https://github.com/jquery/jquery
License:        
Matched files:  LICENSE.txt, package.json
LICENSE.txt:
  Content hash:  683d5d6a7caa50053354cadd3a71744de78e1904
  Attribution:   Copyright JS Foundation and other contributors, https://js.foundation/
  License:       
  Closest licenses:
    MIT similarity:      79.48%
    NCSA similarity:     69.84%
    BSL-1.0 similarity:  59.44%
package.json:
  Confidence:  90.00%
  Matcher:     Licensee::Matchers::NpmBower
  License:     MIT
```

Looks like, the license, which is used by jQuery is not known to GitHub license detector.

**Comment by kborchers** [comment]
@dmethvin This is a much larger change than the one done in ESLint where they removed one minor "ESLint" line at the beginning. That said, in my opinion we can remove lines 3-11 without issue if the team would like to do that. I'm not sure if it will get us close enough for GitHub to be satisfied though. The last lines can not be removed as they define things that may not fall under the MIT license and that section is referenced from our CLA.

**Comment by kborchers** [comment]
Thinking on this a bit more, since the jQuery project is not defining other content that is licensed differently other than the node_modules and external directories, we may be able to remove that but I will need to check with legal and get back to you ASAP.

**Comment by dmethvin** [comment]
Fixed by #4041
